trl/trainer/ppo_trainer.py

```diff
@@ -659,83 +672,90 @@ def step(
 
         timing["time/ppo/forward_pass"] = time.time() - t
 
-        t = time.time()
+        with torch.no_grad():
```

Comment:
Why weren't doing `torch.no_grad()` at the first place? Perhaps we just forgot to add it?

Reply:
Haha I guess we just forgot about it. OAI also did remove the gradient :) (https://github.com/openai/lm-human-preferences/blob/cbfd210bb8b08f6bc5c26878c10984b90f516c66/lm_human_preferences/train_policy.py#L339)

Reply:
I could swear it was there at some point :) thanks for putting it back!